### PR TITLE
Quickstart javascript remix

### PR DESCRIPTION
**Pull request by sixhobbits**
* Please ignore the django-related commits in the first half - they are already in master
* Not sure if types.d.ts file should be in .gitignore as it seems autogenerated?

Depends on https://github.com/ritza-co/fusionauth-quickstart-javascript-remix-web

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205434993506511
  - https://app.asana.com/0/0/1205273462533569

### CODE REVIEW

File: astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+The final page you need is `mysite/app/routes/change.tsx`, with the following code.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-javascript-remix-web/main/complete-application/app/routes/change.tsx"
+ lang="typescript" />
+
+This page has identical Remix and authentication functionality to the Account page and some pure React code that updates the DOM whenever the state changes.
+
+## Run The Application
+
+Start the Remix server.
```
Comment (vcampitelli) [inline astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx:255]:
```suggestion
Start the Remix server from the root `mysite` directory.
```

Or something like that.

**Review by vcampitelli — CHANGES_REQUESTED** [review]

File: astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+Finally, `authStrategy` returns the user's email, decoded from the JWT returned by FusionAuth.
+
+### Routes
+
+First replace the homepage of your app in `mysite/app/routes/_index.tsx` with the following code.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-javascript-remix-web/main/complete-application/app/routes/_index.tsx"
+ lang="typescript" />
+
+If you have not worked with React before, there are a few differences from normal HTML to note:
```
[the remark this answers is not in the document]
Comment (rideam) [reply astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx:191]: switched the paragraphs

File: astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+ lang="typescript" />
+
+This code provides a way for Remix to store cookies that maintain the user's session. (If you'd prefer to use file-based sessions instead of cookie-based, you can read about it in an earlier [blog post about using Remix](/blog/remix-demo).)
+
+Next, create an authentication file in the services folder, `auth.server.ts`, that uses the session provider you just created.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-javascript-remix-web/main/complete-application/app/services/auth.server.ts"
+ lang="typescript" />
+
+In the line `new Authenticator<User>`, you can specify which user details you want to keep in the session. This example uses a single string containing the user's email, but you could define an object with more details or just a user Id. To do so, you would change the definition of `User`, update the object returned in the authentication method (`return token.email`), and update the routes you are about to create to use the new object instead of just an email.
```
[the remark this answers is not in the document]
Comment (rideam) [reply astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx]: removed the extra information

File: astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+Next, you'll set up a basic Remix app. While this guide builds a new Remix project, you can use the same method to integrate your existing project with FusionAuth.
+
+Create the default Remix starter project.
+
+```shell
+npx create-remix@latest mysite
+```
+
+Choose the following options:
```
[the remark this answers is not in the document]
Comment (rideam) [reply astro/src/content/quickstarts/quickstart-javascript-remix-web.mdx:106]: added a line for the user to allow installation

**Review by vcampitelli — APPROVED** [review]

**Comment by vcampitelli** [comment]
@mooreds this is ready for you to review

**Comment by escii** [comment]
looks good, nothing to complain about.
I think this is ready to be published @mooreds

**Review by escii — APPROVED** [review]

**Pull request merged**